.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+name: Release To Maven
+
+# Run workflow on commits to the `main` branch
+on:
+   workflow_dispatch:
+    inputs:
+      Version:
+        description: "Version to be released in format: x.y.z, where x => major version, y => minor version and z => patch version"
+        required: true
+        default: "0.1.0"
+      Title:
+        description: "Title of the release"
+        required: true
+        default: "Improving API developer experience"
+
+jobs:
+  publish:
+    name: Creating release version
+    runs-on: ubuntu-latest
+    environment: Production
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Install Java and Maven
```

Comment:
This should be rename to `Installing Java and Maven setup`

Reply:
Renamed as `Install Java and Maven setup`